doc/tutorial_classification.rst

```diff
@@ -6,4 +6,164 @@
 Tutorial
 ========
 
-TO BE CONTINUED
\ No newline at end of file
+In this tutorial, we compare the prediction sets estimating by Mapieclassifier in a two-dimensional dataset with three labels. The distribution of the data is a bivariate normal with diagonal covariance matrices for each label. 
+
+Throughout this tutorial, we will answer the following questions:
+
+How do the number of class in the prediction sets vary according to the values ​​of alpha with the Conformal Prediction method ?
+
+1. Conformal Prediction method using the softmax score of the true label
+=====================================================================
+
+
```

Comment:
Je commencerais le paragraphe par une phrase un peu plus générique de présentation de la méthode. Je propose un truc du style que tu peux modifier : "We will use MAPIE to estimate a prediction set of several classes such that the probability that the true label of a new test point is included in the prediction set is always higher than the target confidence level 1−alpha."

Reply:
ajouté